src/humidity.jl

```diff
@@ -0,0 +1,63 @@
+"""
+Compute the saturation specific humidity for a single atmospheric level.
+"""
+function get_saturation_specific_humidity(
+    T::Array{NF,2},  # Absolute temperature for a single atmospheric level
+    p::Array{NF,2},  # Normalised surface pressure
+    σ::NF,           # σ of the current atmospheric level
+    M::ModelSetup{NF},
+) where {NF<:AbstractFloat}
+    @unpack nlon, nlat = M.geospectral.geometry
+    @unpack e₀, C₁, C₂, T₀, T₁, T₂ = M.params.humidity_coefs
+
+    Qsat = zeros(nlon, nlat)  # Saturation specific humidity
+
+    @inbounds for j = 1:nlat, i = 1:nlon
```

Comment:
We also need to have the discussion whether for the parametrizations it makes sense to parallelize in the horizontal or in the vertical. Technically all parametrizations should only act on a given atmospheric column, which would motivate a horizontal domain decomposition, however, the arrays are currently layed out in memory as lon,lat,(leapfrog), and then vert such that the dynamical core can act as independent as possible across the vertical. If we want to parallelize single-column style we may want a function instead that takes in a vector which we could obtain from `view(variable,i,j,:)` instead of baking in the lat lon loop